adrs/0014-avro-parsing.md

```diff
@@ -0,0 +1,56 @@
+# 13. AWS OAS parsing
+
+Date: 2024-07-02
+
+
+## Status
+
+Accepted
+
+
+## Context
+
+The Anypoint Platform needs to support AVRO Schemas (because Async 2.x supports AVRO Schemas).
```

Comment:
We should not have any reference to internal products in AMF. AMF is open source.